### PR TITLE
prevent notice, add more data

### PR DESCRIPTION
**Pull request by diggy**
Check if `$category_data[$category_id]` is set, and add all available data to the array (e.g. category description). Tested with PHP 5.2.x.

Example output:

```
array (size=1)
  0 => 
    array (size=9)
      'category_id' => string '19' (length=2)
      'category_site_id' => string '1' (length=1)
      'category_group_id' => string '3' (length=1)
      'category_parent_id' => string '0' (length=1)
      'category_name' => string 'Bulldog' (length=9)
      'category_url_title' => string 'bulldog' (length=9)
      'category_description' => string 'The Bulldog is a medium-sized breed of dog commonly referred to as the English Bulldog or British Bulldog.' (length=106)
      'category_image' => string '{filedir_1}bulldog.jpg' (length=35)
      'category_order' => string '4' (length=1)
```

### CODE REVIEW

File: third_party/wb_category_select/ft.wb_category_select.php [hunk 1]
```diff
@@ -370,12 +370,17 @@ private function _get_category_data($cat_ids)
...
-				'category_name'      => $category_data[$category_id]['cat_name'],
-				'category_url_title' => $category_data[$category_id]['cat_url_title'],
-				'category_image'     => $category_data[$category_id]['cat_image']
-			);
+			if( isset( $category_data[$category_id] ) )
+			{
+				$process = array();
+				foreach ($category_data[$category_id] as $k => $v)
+				{
+					$k = false !== strpos($k,'cat_') ? str_replace('cat_','category_',$k) : 'category_'.$k;
```
Comment (wesbaker) [inline third_party/wb_category_select/ft.wb_category_select.php]: Change the order of the conditional (I'm not a fan of Yoda conditionals) and wrap the conditional in a parentheses.

**Comment by wesbaker** [comment]
Other than my one change it looks good. Bump the version to 1.6 and document the new template tags and add notes to the change log. 

Thanks!

**Comment by diggy** [comment]
@wesbaker done

**Comment by wesbaker** [comment]
Thanks for your additions!

**Comment by diggy** [comment]
@wesbaker You're welcome! Could you draft and publish a 1.6 release here on Github? https://github.com/wesbaker/category_select.ee2_addon/releases

**Comment by wesbaker** [comment]
[All set](https://github.com/wesbaker/category_select.ee2_addon/releases/tag/1.6.0).

**Pull request merged**